tools/targets/PSOC6.py

```diff
@@ -120,13 +120,13 @@ def find_cm0_image(toolchain, resources, elf, hexf, hex_filename):
     from tools.resources import FileType
     hex_files = resources.get_file_paths(FileType.HEX)
     m0hexf = next((f for f in hex_files if os.path.basename(f) == hex_filename), None)
-    if toolchain.target.name.endswith('_PSA'):
+    if toolchain.target.is_PSA_non_secure_target:
```

Comment:
Making sure before the big green button is pressed.

https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L8043

The only current valid target for this is the F`UTURE_SEQUANA_PSA` correct? Not `FUTURE_SEQUANA_M0_PSA`?

Reply:
`find_cm0_image` is only executed for M4 targets (those with `hex_filename` set in targets.json). Therefore, `toolchain.target.name.endswith('_PSA')` is true only for FUTURE_SEQUANA_PSA, hence this change is safe.